src/clib.c

```diff
@@ -56,51 +58,61 @@ main(int argc, const char **argv) {
   debug_init(&debugger, "clib");
 
   // usage
-  if (NULL == argv[1]
-   || 0 == strncmp(argv[1], "-h", 2)
-   || 0 == strncmp(argv[1], "--help", 6)) {
+  if (NULL == argv[1] || 0 == strncmp(argv[1], "-h", 2) || 0 == strncmp(argv[1], "--help", 6))
+  {
     printf("%s\n", usage);
     return 0;
   }
 
   // version
-  if (0 == strncmp(argv[1], "-v", 2)
-   || 0 == strncmp(argv[1], "--version", 9)) {
+  if (0 == strncmp(argv[1], "-v", 2) || 0 == strncmp(argv[1], "--version", 9))
+  {
     printf("%s\n", CLIB_VERSION);
     return 0;
   }
 
   // unknown
-  if (0 == strncmp(argv[1], "--", 2)) {
+  if (0 == strncmp(argv[1], "--", 2))
+  {
     fprintf(stderr, "Unknown option: \"%s\"\n", argv[1]);
     return 1;
   }
 
   // sub-command
   cmd = strdup(argv[1]);
-  if (NULL == cmd) {
+  if (NULL == cmd)
+  {
     fprintf(stderr, "Memory allocation failure\n");
     return 1;
   }
   cmd = trim(cmd);
 
-  if (0 == strcmp(cmd, "help")) {
-    if (argc >= 3) {
+  if (0 == strcmp(cmd, "help"))
+  {
+    if (argc >= 3)
+    {
       free(cmd);
       cmd = strdup(argv[2]);
       args = strdup("--help");
-    } else {
+    }
+    else
+    {
       fprintf(stderr, "Help command required.\n");
       goto cleanup;
     }
-  } else {
-    if (argc >= 3) {
+  }
+  else
+  {
+    if (argc >= 3)
+    {
       args = str_flatten(argv, 2, argc);
-      if (NULL == args) goto cleanup;
+      if (NULL == args)
+        goto cleanup;
     }
   }
   debug(&debugger, "args: %s", args);
 
+  cmd = strcmp(cmd, "i") == 0 ? strdup("install") : cmd; // equality of "i" and "install" like npm
```

Comment:
I don't think we need to allocate memory for a static string. Can we work around the `strdup("install")` here?

Reply:
But, it will have an error. `munmap_chunk(): invalid pointer`.
If i don't put strdup.